tests/test_definitions/column_map_expectations/expect_column_values_to_be_between.json

```diff
@@ -242,6 +269,7 @@
       {
         "title" : "negative_test_with_result_format__boolean_only",
         "exact_match_out": false,
+        "suppress_test_for": ["trino"],
```

Comment:
@kenwade4 How come we are suppressing this test for `Trino`?  Is there something about `Trino` as compared to other `SQL` backends that would preclude us from implementing this expectation?  Thanks.